app/views/self_service/offices/_form.html.erb

```diff
@@ -70,6 +70,14 @@
     <%= f.text_field :telephone_number, class: 't-telephone-number' %>
   <% end %>
 
+  <%= f.form_row :website do %>
+    <%= f.errors_for :website %>
+    <%= f.label :website,
+                Office.human_attribute_name(:website),
+                class: 'form__label-heading' %>
+    <%= f.text_field :website, class: 't-website' %>
```

Comment:
As this PR doesn't have any other changes that refer to it, I'd hazard a guess that this `t-website` class is surplus and can be skipped?